aie_runtime_lib/AIE/tanh.h

```diff
@@ -0,0 +1,55 @@
+//===-  tanh.h - get hyperbolic tangent values based on linear approximation
+//-===//
+//
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// (c) Copyright 2023 Xilinx Inc.
+//
+//
+//===----------------------------------------------------------------------===//
+// This is the implementation of compute hyperbolic tangent values based on
+// linear approximation
+//===----------------------------------------------------------------------===//
+
+#ifndef __TANH__
+#define __TANH__
+
+#include "aie_api/aie.hpp"
+#include <aie_api/aie_adf.hpp>
+#include <aie_api/utils.hpp>
+
+extern float chess_storage(% chess_alignof(v32int8)) tanh_lut_ab[2048];
+extern float chess_storage(% chess_alignof(v32int8)) tanh_lut_cd[2048];
```

Comment:
These can be declared as `extern float *tanh_lut_ab;` or `extern float tanh_lutcd[];`